tests/llm/fixtures/test_ask_holmes/274_silent_degradation_fleet/setup.sh

```
@@ -0,0 +1,235 @@
+    kubectl get pods -n "app-274-$SVC"
+    exit 1
+  fi
+done
+echo "All 24 services ready"
+
+# Verify the degradation trends are fully written to logs
+for PAIR in "shipping-quote:4[89][0-9]ms" "feed-builder:backlog=1[89][0-9][0-9]" "thumbnail-cache:hit_ratio=0.4[0123]"; do
+  SVC="${PAIR%%:*}"
+  PATTERN="${PAIR##*:}"
+  OK=false
+  for i in $(seq 1 90); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)